Add a little note about vi and ex commands in xpg4/6 directories.

Not suitable for upstream

--- vim-9.0.0053/runtime/doc/vim.1
+++ vim-9.0.0053/runtime/doc/vim.1
@@ -29,6 +29,13 @@ vim \- Vi IMproved, a programmer's text
 .B rview
 .B rgvim
 .B rgview
+.PP
+Please note that for the POSIX versions of
+.B vi
+and
+.B ex
+see the man pages for them under /usr/xpg4/man and /usr/xpg6/man directories.
+.br
 .SH DESCRIPTION
 .B Vim
 is a text editor that is upwards compatible to Vi.
